### PR TITLE
Use promises

### PR DESCRIPTION
**Pull request by arthurvr**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 'use strict';
 var childProcess = require('child_process');
 var resolveFrom = require('resolve-from');
+var pify = require('pify');
+var Promise = require('pinkie-promise');
 
-module.exports = function (pth, cb) {
+module.exports = function (pth) {
 	if (typeof pth !== 'string') {
```
Comment (kevva) [inline index.js:8]: Should we use `pth = pth || process.cwd()` here maybe?

Comment (arthurvr) [inline index.js:8]: Happy to change it, do you like that more? I honestly tend to prefer the "long" version these days.

Comment (kevva) [inline index.js:8]: Yeah, I think I like the long version more too. Just raising the question for @sindresorhus.

**Comment by sindresorhus** [comment]
Woot!

**Pull request merged**